hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4136,9 +4137,14 @@ public OzoneConfiguration getConfiguration() {
     return configuration;
   }
 
+  public OmConfig getConfig() {
+    return config;
+  }
+
   @VisibleForTesting
   public void setConfiguration(OzoneConfiguration conf) {
```

Comment:
Make this `synchronized`?

Reply:
I'm against adding `synchronized` here and there.  It does not make the object thread-safe without proper design, but can introduce deadlocks.